packages/apps/frontera/src/routes/organization/src/components/Tabs/panels/AboutPanel/AboutPanel.tsx

```diff
@@ -250,32 +262,100 @@ export const AboutPanel = observer(() => {
             />
           )}
 
-        <div className='flex w-full'>
-          <FormSelect
-            isClearable
-            name='relationship'
-            formId='organization-about'
-            placeholder='Relationship'
-            options={relationshipOptions}
-            leftElement={<HeartHand className='text-gray-500 mr-3' />}
-          />
-        </div>
+        <div className='flex items-center justify-center w-full'>
+          <div className='flex-2'>
+            <Menu>
```

Comment:
Let's extract this to separate component -> this will prevent code duplicate as it will be used in other places as well